src/documentation/pages/components/modal/examples/index.vue

```diff
@@ -0,0 +1,55 @@
+<template lang="pug">
+  div
+    h2 Examples
+    p.-text
+      | Modals are composed of several parts. To render a modal, include a modal backdrop, modal header, modal body,
+      | and modal footer (optional).
+      | By default modals are aligned to the top of the page.
+
+      <Base />
+      <Center />
+      h3 Scrollable
+      p.-text 
+        | Enable scrolling if the height of the modal's content is larger than the modal's content container.
+      <ScrollableRequirements />
+      <ScrollableLongContent />
+      div(v-if="['lumen', 'centurylink'].includes($store.state.themes.theme)")
+        <ModalSubtitle />
+        <Inverse />
+        <NoBorder />
```

Comment:
these should be named ModalSubtitleLumenCenturyLink, InverseLumenCenturyLink etc